app/components/UI/Dialog.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Box, Flex } from 'rebass'
+import { Heading, Button } from 'components/UI'
+import X from 'components/Icon/X'
+
+const Container = styled(Box)`
```

Comment:
You could probably switch from using `Box` to `Card`, which supports the border props. See https://rebassjs.org/Card. Then you could do away with this `Container` component.

Reply:
Agreed